examples/BUILD

```diff
@@ -116,3 +116,23 @@ jsonnet_to_json_test(
     yaml_stream = 1,
     golden = "yaml_stream_golden.yaml",
 )
+
+jsonnet_to_json(
```

Comment:
Please remove this and add a golden file to the jsonnet_to_json_test below.

(See https://github.com/bazelbuild/rules_jsonnet/blob/master/examples/BUILD#L3)